tola/management/commands/loadinitialdata.py

```diff
@@ -9,15 +9,24 @@
 from django.db import transaction, IntegrityError, connection
 
 import factories
+from indicators.models import (Level, Frequency, Indicator, PeriodicTarget,
+                               CollectedData)
 from workflow.models import (
     ROLE_VIEW_ONLY, ROLE_ORGANIZATION_ADMIN, ROLE_PROGRAM_ADMIN,
-    ROLE_PROGRAM_TEAM)
+    ROLE_PROGRAM_TEAM, Organization, Country, TolaUser, Group, Sector,
+    Stakeholder, Milestone, WorkflowLevel1, WorkflowLevel2,
+    WorkflowLevel1Sector, WorkflowTeam)
 
 logger = logging.getLogger(__name__)
 DEFAULT_WORKFLOW_LEVEL_1S = [  # tuple (id, name)
     (3, 'Humanitarian Response to the Syrian Crisis'),
     (6, u'Bildung für sozial benachteiligte Kinder in Deutschland'),
 ]
+DEFAULT_ORG = {
+    'id': 1,
+    'name': 'TolaData',
+}
```

Comment:
We have a settings variable `DEFAULT_ORG` in the `local.py`. You should use it to create the default organization.

Reply:
Done!